src/s_user.c

```diff
@@ -3412,6 +3418,11 @@ m_umode(aClient *cptr, aClient *sptr, int parc, char *parv[])
                             hash_check_watch(sptr, RPL_LOGON);
                         }
                     }
+                    else
+                    {
+                        sendto_one(sptr, ":%s NOTICE %s :*** Notice -- Server has not received hostmask type, please contact SRA.",
```

Comment:
Arfie pointed out that we should change this to something other than "contact SRA" because this could be running on other networks outside of DALnet.

Plus, we don't want every single end-user contacting SRA for this, should something break.  Maybe it should go to opers or a channel or something.  Let's rethinking what makes sense.